event/handler_test.go

```diff
@@ -345,9 +425,12 @@ func TestImagePublishedHandler_Handle(t *testing.T) {
 			mockS3Public.UploadFunc = func(input *s3manager.UploadInput, options ...func(*s3manager.Uploader)) (*s3manager.UploadOutput, error) {
 				return &s3manager.UploadOutput{}, nil
 			}
-			mockImageAPIFail := &mock.ImageAPIClientMock{
+			mockVariantAPIFail := &mock.ImageAPIClientMock{
```

Comment:
Previous name of var fine
```suggestion
			mockImageAPIFail := &mock.ImageAPIClientMock{
```